.github/workflows/draft_github_release.yaml

```diff
@@ -0,0 +1,85 @@
+name: Draft GitHub Release
+
+# Runs when a new tag is created that looks like a version number.
+#
+# Creates a draft release on GitHub with the latest release notes, then chains
+# to the build_and_test workflow and the publish_github_release workflow.
+#
+# Collectively, this will build and tests on all OSes, attach release artifacts
+# to the release, and fully publish the release.
+#
+# Publishing the release then triggers additional workflows for NPM, Docker
+# Hub, and GitHub Pages.
+#
+# Can also be run manually for debugging purposes.
+on:
+  push:
+    tags:
+      - "v*.*"
+  # For manual debugging:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "An existing tag to release."
+        required: True
+
+jobs:
+  draft_release:
+    name: Draft GitHub release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: src
+
+      - name: Compute ref
+        # We could be building from a workflow dispatch (manual run)
+        # or from a pushed tag.  If triggered from a pushed tag, we would like
+        # to strip refs/tags/ off of the incoming ref and just use the tag
+        # name.  Subsequent steps can refer to $TARGET_REF to determine the
+        # correct ref in all cases.
+        run: |
+          echo "TARGET_REF=${{ github.event.inputs.tag || github.ref }}" | \
+              sed -e 's@refs/tags/@@' >> $GITHUB_ENV
```

Comment:
Can't you just do `TARGET_REF=refs/tags/{{...}}`?

Reply:
No, you've got it backwards.  I'm stripping off the `refs/tags/` part from the input to get the tag name only.  I suppose this needs a more local comment.